website/pages/docs/internals/security.mdx

```diff
@@ -118,7 +118,8 @@ client), the highest access level permitted is used. For example, if the
 "engineering" policy permits read/update access to the "eng/" path, and the
 "ops" policy permits read access to the "ops/" path, then the user gets the
 union of those. Policy is matched using the most specific defined policy, which
-may be an exact match or the longest-prefix match glob pattern.
+may be an exact match or the longest-prefix match glob pattern. See
```

Comment:
It's not always the longest-prefix match right? If it's not, we shouldn't say that part here.

Reply:
I do not know. I'll have to refer to @mgritter

Reply:
@mgritter do you think it would be more correct to say something like:

"which may be an exact match or will otherwise follow the rules described in the [Policy Syntax]...

?

Reply:
Something like that would be better, although it's unfortunate that we can't emphasize the common case that longest-prefix is how it works as long as you're only doing globs.

I don't like this whole sentence, though, "policy is matched using the most specific defined policy" means what?  I think we need to clarify somehow that:
* If the same pattern appears in multiple policies, we take the union of the capabilities
* If different patterns appear in the applicable policies, we take only the highest-priority match from those policies

Something like "The policy rules that Vault applies are determined by the most-specific match available, using the priority rules described [blah]."